cmd/launcher/main.go

```diff
@@ -16,10 +16,10 @@ import (
 )
 
 func main() {
-	cmd.Exit(runLaunch())
+	cmd.Exit(RunLaunch())
 }
 
-func runLaunch() error {
+func RunLaunch() error {
```

Comment:
Relaying contents from Slack conversation - making this public won't help, since it's still in `main`. @samj1912 to update.